code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -858,11 +858,18 @@
 	if(cached_gases[/datum/gas/bz][MOLES] - consumed_amount < 0)
 		return NO_REACTION
 	if(cached_gases[/datum/gas/bz][MOLES] < 30)
+		ASSERT_GAS(/datum/gas/nitrogen, air)
+		ASSERT_GAS(/datum/gas/helium, air)
+		ASSERT_GAS(/datum/gas/plasma, air)
+		cached_gases[/datum/gas/nitrogen][MOLES] += consumed_amount * 0.4
+		cached_gases[/datum/gas/helium][MOLES] += consumed_amount * 1.6
+		cached_gases[/datum/gas/plasma][MOLES] += consumed_amount * 0.8
 		cached_gases[/datum/gas/bz][MOLES] -= consumed_amount
+		energy_released += consumed_amount * 60000
```

Comment:
this seems like a lot, how fast the temperature rises?

Reply:
Depends on the heat capacity really. It was intended to be used with a heat bank like a bunch of plasma to reduce the rate it increases. The massive heat capacity increase will limit it to go up to 328K to 340K from 260K to 280K.